test/online/fetch_test.rb

```diff
@@ -81,6 +81,14 @@ def test_fetch_over_ssh_with_credentials
       })
     end
 
+    def test_fetch_over_ssh_with_credentials_from_memory
+      @repo.remotes.create("origin", ENV['GITTEST_REMOTE_SSH_URL'])
+
+      @repo.fetch("origin", {
+          credentials: ssh_key_credential_from_memory
```

Comment:
This will work as long as we keep compiling against libssh2 which itself is built aginst openssl, but otherwise we'll start seeing errors in test.